### PR TITLE
test: update env files

### PR DESCRIPTION
**Pull request by PraveenAsokan**
## What has been achieved in this pull request?

- [x] Achievement completed

## Related issues

#

## Checklist before merging
- [ ] If it is a core feature, I have added thorough tests.
- [ ] This PR stays within scope of the related issue.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+tag=@card,@cta,@detail,@form,@image,@icon,@inputter,@cake
```
Comment (MekalaNagarajan-Centrica) [inline .env:1]:
```suggestion
tags=@card, @cta, @detail, @form, @image, @icon, @inputter, @cake
```

File: config/baseConfig.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+	"env": {
+		"TAGS": "@form or @inputter or @cake"
```
Comment (MekalaNagarajan-Centrica) [inline config/baseConfig.json:3]:
```suggestion
		"tags": "@form or @inputter or @cake"
```

File: config/baseConfig.json
Comment (MekalaNagarajan-Centrica) [inline config/baseConfig.json:1]: change the filename to `base-config`.json

File: config/fileReadWrite.js
Comment (MekalaNagarajan-Centrica) [inline config/fileReadWrite.js:1]: rename the file to `json-handler.js`

File: config/fileReadWrite.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+
+// Function to read and any JSON file.
+
+function jsonReader(filePath, cb) {
```
Comment (MekalaNagarajan-Centrica) [inline config/fileReadWrite.js:5]:
```suggestion
function readJSON(filePath, cb) {
```

File: config/set_env.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const jsonReader = require('./fileReadWrite');
+
+let tag;
```
Comment (MekalaNagarajan-Centrica) [inline config/set_env.js:4]:
```suggestion
let tag = '';
```

File: config/set_env.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+if (process.env.tag.includes(',')) {
+  tag = process.env.tag.split(',').join(" or ");
+}else if(process.env.tag){
+  tag = process.env.tag
+}else {
+  tag = ""
+}
```
Comment (MekalaNagarajan-Centrica) [inline config/set_env.js:12]:
```suggestion
const envTags = process.env.tag;
if (envTags) {
  tag = envTags.includes(',') ? envTags.split(',').join(' or ') : envTags;
}
```

File: config/set_env.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+
+//Updating the BaseConfig.JSON file and Updating the ENV and Branch ID as per the input from runtime. 
+
+jsonReader("./config/baseConfig.json", (err, template) => {
+  if (err) {
+    console.log("Error reading file:", err);
+    return;
+  }
+  template.env.TAGS = tag;
```
Comment (MekalaNagarajan-Centrica) [inline config/set_env.js:22]:
```suggestion
  template.env.tags = tag;
```

File: config/set_env.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+
+  fs.writeFile("./config/baseConfig.json", JSON.stringify(template, null, "\t"), err => {
+    if (err) console.log("Error writing file:", err);
+  });
+  console.log(`==>***** Tag is set as expected *****<==`);
+});
```
Comment (MekalaNagarajan-Centrica) [inline config/set_env.js:29]: move these code block to `json-handler.js`

File: package.json [hunk 8]
```diff
@@ -29,6 +29,7 @@
     "test:percy:custom": "start-server-and-test 'npm run dev --no-open' 8000 'percy snapshot snapshots.yml'",
     "test:ember": "start-server-and-test 'npm run open:ember' 4200 './node_modules/.bin/cypress-tags run -e TAGS='@cake' --browser chrome'",
     "open:ember": "cd examples/ember-app && npm install && npm run build-muon && npm run start",
+    "setEnv": "env-cmd node config/set_env.js",
```
Comment (MekalaNagarajan-Centrica) [inline package.json:32]:
```suggestion
    "set-env": "env-cmd node config/set_env.js",
```

File: config/set_env.js
Comment (MekalaNagarajan-Centrica) [inline config/set_env.js:1]: rename the file to `environment.js`

**Review by andij — APPROVED** [review]

**Pull request merged**